### PR TITLE
Scripts for creating an exercise

### PR DESCRIPTION
**Pull request by glennj**
Creation script
* calls `configlet create`
* creates stubs for the proof, spec and stub files
* creates the package.wren file and the LICENSE

### CODE REVIEW

**Comment by joshgoebel** [comment]
Love this idea!

File: bin/create-exercise [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+sed -e "s/%{slug}/${slug}/g" \
+    -e "s/%{PascalSlug}/${pascal}/g" <<'__PROOF_WREN__' > "${exDir}/.meta/proof.ci.wren"
+class %{PascalSlug} {
+  // implement me!
+}
+__PROOF_WREN__
+
+############################################################
+
+read -p "What's your github handle? "
```
Comment (joshgoebel) [inline bin/create-exercise:115]:
```suggestion
read -p "What's your GitHub handle? "
```

File: bin/create-exercise [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+bin/configlet create --practice-exercise "${slug}"
+
+shDir=./exercises/shared/new-exercise-templates
+exDir=./exercises/practice/${slug}
+
+############################################################
+cat <<'__LICENSE__' > "${exDir}/LICENSE"
+MIT License
+
+Copyright (c) 2023 Exercism
```
Comment (joshgoebel) [inline bin/create-exercise:35]: 2024?

File: bin/create-exercise [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+import "./%{slug}" for %{PascalSlug}
+import "wren-testie/testie" for Testie, Expect
+
+Testie.test("%{PascalSlug}") { |do, skip|
+  do.test("first test") {
+    Expect.value(%{PascalSlug}.isPangram("")).toEqual(false)
+  }
+
+  skip.test("subsequent tests") {
+    Expect.value(%{PascalSlug}.isPangram("abcdefghijklmnopqrstuvwxyz")).toEqual(true)
```
Comment (joshgoebel) [inline bin/create-exercise:96]: Should we make these more generic rather than reference pangram exercise?

**Review by joshgoebel — DISMISSED** [review]

**Comment by glennj** [comment]
@joshgoebel ping.

**Comment by glennj** [comment]
@ErikSchierboom if you please

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**